### PR TITLE
fileIO to dataIO conversions

### PR DESCRIPTION
**Pull request by Sitryk**

### CODE REVIEW

File: adventure/adventure.py [hunk 1]
```diff
@@ -3273,18 +3273,18 @@ def check_files():
 
     files = {'teams.json' : {}, 'settings.json' : {}}
     for file, default in files.items():
-        if not os.path.isfile(base_path + file):
```
Comment (Chovin) [inline adventure/adventure.py]: there are some functionalilty choices you're sprinking in this PR as well.
This makes it slower for me to check this large PR.
If you had made smaller PRs that encompassed each file, I'd be able to approve them iteratively and have those changes in master quicker.

Comment (Sitryk) [reply adventure/adventure.py]: Ah sorry, I'll fix that now
I thought it would work the same way

File: economytrickle/economytrickle.py [hunk 2]
```diff
@@ -174,7 +174,8 @@ def __init__(self, bot):
             if self.settings[sid]["TRICKLE_BOT"]:
                 await self.bot.say("I will now get currency trickled to me.")
             else:
-                await self.bot.say("I will stop getting currency trickled to me.")
+                await self.bot.say("I will stop getting currency "
+                                   "trickled to me.")
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: what's with all these formatting changes in this PR also?

File: economytrickle/economytrickle.py [hunk 3]
```diff
@@ -192,7 +193,8 @@ def __init__(self, bot):
             await self.bot.say("Timeout interval must be more than 0")
             return
         self.settings[sid]["ACTIVE_TIMEOUT"] = minutes
-        await self.bot.say("Active user timeout is now: " + str(self.settings[sid]["ACTIVE_TIMEOUT"]))
+        await self.bot.say("Active user timeout is now: " +
+                           str(self.settings[sid]["ACTIVE_TIMEOUT"]))
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: formatting changes should be in a separate PR

File: economytrickle/economytrickle.py [hunk 4]
```diff
@@ -204,7 +206,9 @@ def __init__(self, bot):
             await self.bot.say("```Warning: With an interval this low, "
                                "a trickle will occur after every message```")
         self.settings[sid]["PAYOUT_INTERVAL"] = minutes
-        await self.bot.say("Payout interval is now: " + str(self.settings[sid]["PAYOUT_INTERVAL"]))
+
+        await self.bot.say("Payout interval is now: " +
+                           str(self.settings[sid]["PAYOUT_INTERVAL"]))
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: formatting changes should be in a separate PR

File: economytrickle/economytrickle.py [hunk 5]
```diff
@@ -218,7 +222,8 @@ def __init__(self, bot):
                                "users you have. This will discourage "
                                "conversations.```")
         self.settings[sid]["PAYOUT_PER_ACTIVE"] = amt
-        await self.bot.say("Base payout per active user is now: " + str(self.settings[sid]["PAYOUT_PER_ACTIVE"]))
+        await self.bot.say("Base payout per active user is now: " +
+                           str(self.settings[sid]["PAYOUT_PER_ACTIVE"]))
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: formatting changes should be in a separate PR

File: economytrickle/economytrickle.py [hunk 6]
```diff
@@ -234,7 +239,8 @@ def __init__(self, bot):
                                "unless you want to discourage conversations"
                                "```")
         self.settings[sid]["NEW_ACTIVE_BONUS"] = amt
-        await self.bot.say("Bonus per new active user is now: " + str(self.settings[sid]["NEW_ACTIVE_BONUS"]))
+        await self.bot.say("Bonus per new active user is now: " +
+                           str(self.settings[sid]["NEW_ACTIVE_BONUS"]))
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: formatting changes should be in a separate PR

File: economytrickle/economytrickle.py [hunk 7]
```diff
@@ -250,7 +256,8 @@ def __init__(self, bot):
                                "trickle. With a negative leak, the bonus pot "
                                "will grow each time a trickle occurs.```")
         self.settings[sid]["ACTIVE_BONUS_DEFLATE"] = amt
-        await self.bot.say("Bonus pot leak is now: " + str(self.settings[sid]["ACTIVE_BONUS_DEFLATE"]))
+        await self.bot.say("Bonus pot leak is now: " +
+                           str(self.settings[sid]["ACTIVE_BONUS_DEFLATE"]))
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: formatting changes should be in a separate PR

File: economytrickle/economytrickle.py [hunk 8]
```diff
@@ -266,7 +273,8 @@ def __init__(self, bot):
                                "You might as well just unload "
                                "cogs.economytrickle```")
         self.settings[sid]["CHANCE_TO_PAYOUT"] = percentage
-        await self.bot.say("Successful trickle chance is now: " + str(self.settings[sid]["CHANCE_TO_PAYOUT"]))
+        await self.bot.say("Successful trickle chance is now: " +
+                           str(self.settings[sid]["CHANCE_TO_PAYOUT"]))
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: formatting changes should be in a separate PR

File: economytrickle/economytrickle.py [hunk 9]
```diff
@@ -311,11 +319,21 @@ def __init__(self, bot):
...
+        if not toggle:
+            return
+        if (toggle is True and
+                message.channel.id not in self.settings[sid]['CHANNELS']):
+            return
+        current_user = self.currentUser.get(sid, None)
+        if ((self.settings[sid]["TRICKLE_BOT"] or
+             message.author.id != self.bot.user.id) and
+                current_user != message.author.id):
+            # print("----Trickle----")
```
Comment (Chovin) [inline economytrickle/economytrickle.py]: these changes are out of scope of the PR

File: snacktime/snacktime.py [hunk 10]
```diff
@@ -39,7 +39,6 @@ def __init__(self,bot):
         self.repeatMissedSnacktimes = dataIO.load_json("data/snacktime/repeatMissedSnacktimes.json")
         self.channels = dataIO.load_json("data/snacktime/channels.json")
         self.settings = dataIO.load_json("data/snacktime/settings.json")
-        self.defaultSettings = {"EVENT_START_DELAY" : 1800, "EVENT_START_DELAY_VARIANCE" : 900, "SNACK_DURATION" : 240, "SNACK_DURATION_VARIANCE" : 120, "MSGS_BEFORE_EVENT" : 8, "SNACK_AMOUNT" : 200}
```
Comment (Chovin) [inline snacktime/snacktime.py]: why was this deleted?

File: snacktime/snacktime.py [hunk 11]
```diff
@@ -121,7 +120,7 @@ def __init__(self,bot):
         """snack stuff"""
         scid = ctx.message.server.id+"-"+ctx.message.channel.id
         if self.settings.get(scid, None) == None:
-            self.settings[scid] = self.defaultSettings
+            self.settings[scid] = deepcopy(DEFAULT_SETTINGS)
```
Comment (Chovin) [inline snacktime/snacktime.py]: these changes are out of scope of the PR

**Review by Chovin — CHANGES_REQUESTED** [review]
